src/Lie_algebra/Lie_algebra_interface.jl

```diff
@@ -0,0 +1,86 @@
+
+"""
+    LieAlgebra{𝔽, G} <: AbstractManifold{𝔽}
+
+Represent the Lie Algebra ``$(_math(:𝔤))``, that is a ``𝔽``vector space with an associated
+[`lie_bracket`](@ref) ``[⋅,⋅]: $(_math(:𝔤))×$(_math(:𝔤)) → $(_math(:𝔤))`` which fulfills
+
+1. ``[X,X] = 0`` for all ``X ∈ $(_math(:𝔤))``
+2. The Jacobi identity holds ``[X, [Y,Z]] = [[X,Y],Z] = [Y, [X,Z]]`` holds for all ``X, Y, Z ∈ $(_math(:𝔤))``.
+
+The Lie algebras considered here are those related to a [`LieGroup`](@ref) ``$(_math(:G))``,
+namely the tangent space the tangent space ``T_{$(_math(:e))}$(_math(:G))`` at the [`Identity`](@ref),
+this is internally just a `const` of the corresponding $(_link(:TangentSpace)).
+
+# Constructor
+
+    LieAlgebra(G::LieGroup)
+
+Return the Lie Algebra belonging to the [`LieGroup`](@ref) `G`.
+"""
+const LieAlgebra{𝔽,G,I} = ManifoldsBase.Fiber{
+    𝔽,ManifoldsBase.TangentSpaceType,G,I
+} where {𝔽,G<:LieGroup{𝔽},I<:Identity}
```

Comment:
```suggestion
const LieAlgebra{𝔽,G} = ManifoldsBase.Fiber{
    𝔽,ManifoldsBase.TangentSpaceType,G,Identity{O}
} where {𝔽,O<:AbstractGroupOperation,G<:LieGroup{𝔽,O}}
```
This should simplify things a bit.

Reply:
Hm, I think this broke something. I'd have to check.

Reply:
I've figured out a slightly different approach and committed it.